### PR TITLE
Add 'prepare-release.sh' script

### PR DESCRIPTION
**Pull request by e0ne**
This script updates chart version and creates helm package.

Related: #131

### CODE REVIEW

File: scripts/releases/update-gh-pages.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+this=`basename $0`
+
+usage () {
+cat << EOF
+Usage: $this [-h] [-a] <helm-chart>
+
+Options:
+  -h         show this help and exit
+  -p REMOTE  do git push to remote repo
```
Comment (moshe010) [inline scripts/releases/update-gh-pages.sh:26]: update usage

File: scripts/releases/update-gh-pages.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o pipefail
+
+this=`basename $0`
+
+usage () {
+cat << EOF
+Usage: $this [-h] [-a] <helm-chart>
```
Comment (moshe010) [inline scripts/releases/update-gh-pages.sh:22]: what is the -a? I don't see it used

File: scripts/releases/prepare-release.sh [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+# Copyright 2020 NVIDIA
```
[the remark this answers is not in the document]
Comment (e0ne) [reply scripts/releases/prepare-release.sh:2]: I hope, sometime I'll do it without a mistake or create a CI check. Thank you for the review!

**Review by moshe010 — COMMENTED** [review]
Please update https://github.com/Mellanox/network-operator/blob/master/.github/ISSUE_TEMPLATE/new-release.md the as well

**Review by moshe010 — APPROVED** [review]

**Pull request merged**